docker/api/build.py

```diff
@@ -290,7 +300,12 @@ def prune_builds(self):
                 If the server returns an error.
         """
         url = self._url("/build/prune")
-        return self._result(self._post(url), True)
+        params = {}
+        if filters is not None:
+            params['filters'] = utils.convert_filters(filters)
+        if keep_storage is not None:
+            params['keep-storage'] = keep_storage
+        return self._result(self._post(url, params=params), True)
```

Comment:
These args were added in API v1.39 - typically, we add a check for them, e.g.
```python
if utils.version_lt(self._version, '1.39'):
    raise errors.InvalidVersion(
        '`keep-storage` arg only available for API version > 1.38'
    )
```

(if you look for `utils.version_lt` or `utils.version_gte` you'll see a bunch of examples - they're not 100% consistent so do whatever makes sense here, e.g. I'm fine if there's a single check if ANY of `filters` / `keep_storage` / `any` are not `None` since they were all added together in the same API version)

Reply:
Done!